x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -377,6 +377,22 @@ export const PolicyList = React.memo(() => {
     [services.application, handleDeleteOnClick, formatUrl, search]
   );
 
+  const bodyContent = useMemo(() => {
```

Comment:
this one feels a bit scarier to change, but `policy_list` isn't current accessible.  Did you test it out?